ignite/handlers/lr_finder.py

```diff
@@ -137,6 +137,17 @@ def _reset(self, trainer: Engine) -> None:
     def _log_lr_and_loss(self, trainer: Engine, output_transform: Callable, smooth_f: float, diverge_th: float) -> None:
         output = trainer.state.output
         loss = output_transform(output)
+        if not isinstance(loss, float):
+            if isinstance(loss, torch.Tensor) and (
+                (loss.ndimension() == 0) or (loss.ndimension() == 1 and len(loss) == 1)
+            ):
+                loss = loss.item()
+            else:
```

Comment:
This looks a bit bulky but I have no idea on how to simplify it more than one line.